src/styles/utils/variables.scss

```diff
@@ -0,0 +1,14 @@
+$container-day-width: 100px;
+$container-day-height: 100px;
+$day-gap: 1px;
+$calendar-width: 7 * $container-day-width + 20 + $day-gap * 6;
+$margin-fist-child: 6 * $container-day-width + $day-gap * 5;
+$calendar-start-day: (
+  mon:0,
+  tue:1,
+  wed:2,
+  thu:3,
+  fri:4,
+  sat:5,
+  sun:6,
+);
```

Comment:
This is correct, but I can't find where u use this. Maybe you need one more loop in main file, to use this for modificator.